app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
dấu ```||=``` dùng để làm gì nhỉ

Reply:
nó giống toán tử gán ạ. Dùng <=> x =x || y 
nếu @current_user = nil thì gán id đc find_by session_id
mục đích gán id của session đầu tiên khi đăng nhập

Reply:
a ||= b means assign b to a if a is null or undefined or false